meta-resin-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -1,8 +1,15 @@
-#!/bin/bash
+#!/bin/sh
 
 set -o errexit
 set -o nounset
 
+finish() {
+    # Make all files owned by the build system
+    chown -R "$USER_ID:$USER_GID" "$DATA_VOLUME"
+}
+
+trap finish EXIT
```

Comment:
is it enough to just catch `EXIT`? I am not familiar enough whether it would catch any other ways that the things can finish, and whether things can finish any other way here (sigterm, sigkill)

Reply:
It does catch SIGTERM. You can't trap sigkill.

Reply:
:+1: